crates/nu-protocol/src/pipeline_data.rs

```diff
@@ -671,9 +672,11 @@ impl PipelineData {
         to_stderr: bool,
     ) -> Result<i64, ShellError> {
         for item in self {
+            let mut is_err = false;
             let mut out = if let Value::Error { error } = item {
                 let working_set = StateWorkingSet::new(engine_state);
-
+                // Value::Errors must always go to stderr, not stdout.
+                is_err = true;
```

Comment:
Does anyone have an opinion about these lines? I don't know why Value::Errors seemingly were allowed to go to `stdout`, but that seemed wrong to me (and I needed to make this change to make a test pass, due to the aforementioned business about creating but not returning Value::Errors in `follow_cell_path_helper()`.

Reply:
I think it makes sense to force errors to stderr.